### ISSUE TITLE
Was a back-end for org-babel ever implemented?

### ISSUE THREAD

**Issue by xeijin** [jschaf/powershell.el#21]
I want to pass org-mode table references as variables into powershell `src`, but I get an error when I try to execute a code block, presumably because the necessary backend (e.g. ob-powershell) isn't implemented. Is that right?

**Comment by jschaf** [comment]
I'm not sure.  I mostly just moved the code to GitHub and applied some cleanup.  It hasn't been touched in a whilel

**Comment by xeijin** [comment]
Yes it doesn't unfortunately, read on another thread that you no longer use Powershell so that's fair enough, I will learn to live without :)
